src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -480,10 +480,11 @@
     <value>Split pane vertically</value>
   </data>
   <data name="CopyTextCommandKey" xml:space="preserve">
-    <value>Copy text</value>
+    <value>Copy text {0} {1}</value>
+    <comment>{0} will be replaced with SingleLine value. {1} will be replaced with formats</comment>

```

Comment:
this only really works in english and isn't friendly to localization

Reply:
Could I get a bit of help here? It seems fine to me but I don't know what you're looking for. I imagine the localization team would be able to fill in {0} with text for single line, and {1} with text for copyFormatting (or move it all around if necessary).

Reply:
chatted o*line